automation/test.sh

```diff
@@ -38,18 +38,18 @@ if [[ $TARGET =~ openshift-.* ]]; then
   # of old vms does not go fast enough they run out of memory.
   # To still allow continuing with the tests, give more memory in CI.
   export KUBEVIRT_MEMORY_SIZE=6144M
-  if [[ $TARGET =~ .*-crio-.* ]]; then
+  if [[ $TARGET =~ .*-crio.* ]]; then
```

Comment:
you can remove all `.*` from the end, but it does not critical

Reply:
Done